.github/workflows/knative-style.yaml

```diff
@@ -60,3 +60,11 @@ jobs:
               exit 1
           fi
           echo "${{ github.repository }} is formatted correctly."
+      
+      - name: Language
+        shell: bash
+        run: |
+          changed=$(git diff --name-only HEAD HEAD~1 | grep -v '/vendor\|/third-party\|.github\|/hack\|go.sum\|go.mod\|.md')
```

Comment:
pretty much same question (and probably same answer :)), but just to check: why exclude `/hack`? seems quite easy to imagine e.g. a "whitelist" reference slipping in to one of the scripts (and AlexJS won't catch anything there).

Reply:
Indeed yes.

Imma rework it a bit to check hack for everything but not master maybe :thinking:

Reply:
This is just to prevent backsliding, right?
```
# Avoid reusing painful and divisive language until GitHub provides better tools
$MAIN_BRANCH=master
```

Reply:
No longer relevant, checking MD now too.